main.py

```diff
@@ -1,7 +1,9 @@
-import gui as GUI
+import sys
+sys.path.append("./gui")
```

Comment:
I've moved and separated all the gui classes into the directory called gui (obviously :D ) , but I am not sure if this way is a good way to import a python file from another directory. What do you think?